src/components/App/App.module.css

```diff
@@ -0,0 +1,9 @@
+.wrap {
+    display: block;
+    margin: auto;
+    width: 300px;
+}
+
+.title {
+    font-family: 'Montserrat', sans-serif;
+}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/WebHeroSchool/todo_test/pullRequest?prid=5688966)